documentation/modules/ROOT/pages/accessing-hosted-cluster.adoc

```diff
@@ -229,6 +231,9 @@ hosted   4.13.0    hosted-admin-kubeconfig   Completed   True        False
 [#accessing-hostedcluster-ocp-console]
 == Accessing the Hosted Cluster using the OCP Console
 
+(how in a net diagram you access the cluster? is the api traffic goes to management cluster? probably yes.
+ Maybe worth point out the ".apps." part.
```

Comment:
when someone access an api ocp cluster, he expects to reach .api.cluster.com not x.apps.cluster.com. For hypershift this changed because control is an app in management cluster

Reply:
That's wrong. Hosted Clusters still use the api.cluster.com. That DNS record must point to the LB Service IP or to the nodes IP if you're using a NodePort to expose the API.